### ISSUE TITLE
Internal Server Error (500) when using Owin ASP.NET WebAPI SelfHost

### ISSUE THREAD

**Issue by akamyshanov** [rbeauchamp/Swashbuckle.OData#77]
Looking into the problem, might be related to https://github.com/rbeauchamp/Swashbuckle.OData/issues/72

Minimal project that reproduces this: [Swashbuckle.OData.BugSample.zip](https://github.com/rbeauchamp/Swashbuckle.OData/files/165663/Swashbuckle.OData.BugSample.zip)

This does not look right (works ok if only one ODataController is left): 
<img width="549" alt="2016-03-09 at 19 06" src="https://cloud.githubusercontent.com/assets/1358330/13643607/5ebfc6d2-e62a-11e5-9722-2822b496ff7f.png">

**Comment by akamyshanov** [comment]
This solves it: 
```
docsConfig.ResolveConflictingActions(enumerable => enumerable.First());`
```

The cause of the problem is that the `/odata/` route is generated for each `ODataController`, that's why `ResolveConflictingActions` fixes it. I'm not sure that it's a bug but it definitely looks like one, don't think that this `/odata/` route should be present in the first place.

If it helps, here is the stacktrace of the exception without `ResloveConflictingActions`:
```
   at Swashbuckle.OData.ODataSwaggerProvider.CreatePathItem(IEnumerable`1 apiDescriptions, SchemaRegistry schemaRegistry) in C:\Workspace\Projects\Swashbuckle.OData\Swashbuckle.OData\ODataSwaggerProvider.cs:line 144
   at Swashbuckle.OData.ODataSwaggerProvider.<>c__DisplayClass4_0.<GetSwagger>b__3(IGrouping`2 group) in C:\Workspace\Projects\Swashbuckle.OData\Swashbuckle.OData\ODataSwaggerProvider.cs:line 72
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector)
   at Swashbuckle.OData.ODataSwaggerProvider.GetSwagger(String rootUrl, String apiVersion) in C:\Workspace\Projects\Swashbuckle.OData\Swashbuckle.OData\ODataSwaggerProvider.cs:line 68
   at Swashbuckle.Application.SwaggerDocsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken) in C:\Workspace\Projects\Swashbuckle\Swashbuckle.Core\Application\SwaggerDocsHandler.cs:line 32
   at System.Net.Http.HttpMessageInvoker.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Web.Http.Dispatcher.HttpRoutingDispatcher.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.DelegatingHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Web.Http.HttpServer.<SendAsync>d__0.MoveNext()
```

**Comment by rbeauchamp** [comment]
ok. thanks for the bug sample code. that helps!

**Comment by rbeauchamp-lendingtree** [comment]
Fixed in v2.18.0
